app/components/layers/useCapitalProjectsLayer.client.tsx

```diff
@@ -108,6 +115,15 @@ export function useCapitalProjectsLayer(opts?: { visible?: boolean }) {
     getPointRadius: 5,
     getLineColor: [255, 255, 255, 255],
     getLineWidth: 1,
+    onHover: (data) => {
+      const managingCodeCapitalProjectId =
+        data.object?.properties?.managingCodeCapitalProjectId;
+      if (data.index === -1) {
+        setHoveredOverProject(null);
+      } else if (managingCodeCapitalProjectId) {
```

Comment:
Truthy check will fail when value is 0, empty string, or some other falsy value. Prefer explicit checks, like a negation of undefined/null. 

```
managingCodeCapitalProjectId !== undefined
```